graph.py

```diff
@@ -171,7 +171,7 @@ def main(todo_file, done_file, cutoffDays = 7):
         sys.exit(1)
 
     if os.path.isfile(sys.argv[1]) and os.path.isfile(sys.argv[2]):
-        if len(sys.argv) is 4:
+        if len(sys.argv) == 4: # Avoid SyntaxWarning in Python 3.8
```

Comment:
```suggestion
        if len(sys.argv) == 4:
```
Hi @andycyca,
Thanks for the pull request! Can you remove the comment? When reading the code (outside of this PR) it is not obvious what it means.

Reply:
Sure thing! I'm still new to pull requests, but I think it's done: <337b1b1f8bdd6d3eacd157aea7ff28589f6da246>